### PR TITLE
Minimize CSS and JS in non-hot mode

### PR DESCRIPTION
**Pull request by charypar**
Turns out that was pretty easy.

@richard-still-ft @phamann @ironsidevsquincy

### CODE REVIEW

**Comment by charypar** [comment]
Oh, hold on, JS isn't minified either! :( I'll fix that too

File: webpack.config.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ const webpack = require('webpack');
 const BowerWebpackPlugin = require('bower-webpack-plugin');
 const ExtractTextPlugin = require('extract-text-webpack-plugin');
 
-const plugins = [
+var plugins = [
```
Comment (ironsidevsquincy) [inline webpack.config.js]: `let`?

File: webpack.config.js [hunk 2]
```diff
@@ -15,13 +15,19 @@ const plugins = [
...
+		new webpack.optimize.UglifyJsPlugin({
+			compress: {
+        warnings: false
+    	}
+		}),
 		new webpack.optimize.DedupePlugin()
 	]);
 }
 
+console.log(plugins);
```
Comment (ironsidevsquincy) [inline webpack.config.js]: needed?

File: webpack.config.js [hunk 3]
```diff
@@ -15,13 +15,19 @@ const plugins = [
...
 ];
 
-if(process.env.NODE_ENV === 'production') {
+if(process.env.NODE_ENV === 'production' || process.env.NODE_ENV === 'branch') {
 	plugins = plugins.concat([
-		new webpack.optimize.UglifyJsPlugin(),
+		new webpack.optimize.UglifyJsPlugin({
+			compress: {
+        warnings: false
+    	}
```
Comment (ironsidevsquincy) [inline webpack.config.js]: indentation

File: webpack.config.js (hunk 1 above)
Comment (charypar) [inline webpack.config.js]: Apparently not allowed :(

File: webpack.config.js (hunk 2 above)
Comment (charypar) [inline webpack.config.js]: No... :facepalm:

**Comment by ironsidevsquincy** [comment]
:+1:

**Pull request merged**